beacon-chain/core/state/transition.go

```diff
@@ -616,3 +616,40 @@ func ProcessEpochPrecompute(ctx context.Context, state *pb.BeaconState) (*pb.Bea
 	}
 	return state, nil
 }
+
+// computesStateRoot computes the state root of the block without verifying proposer signature
+// and randao.
+func computesStateRoot(
```

Comment:
Should this be named computeStateRootNoVerify?

Reply:
I mean it's `computeStateRoot`, by default `computeStateRoot` does not verify any signature, it's purpose is to pass the state root to proposer to sign